tasks/setup.yaml

```diff
@@ -12,3 +16,26 @@ tasks:
       - description: "Initialize the cluster with Zarf"
         # renovate: datasource=github-tags depName=defenseunicorns/init versioning=semver
         cmd: "uds zarf package deploy oci://defenseunicorns/init:v0.32.1 --confirm"
+
+  - name: registry-login
+    inputs:
+      registry:
+        default: registry1.dso.mil
+      registry-username:
+      registry-password:
+      registry-retry-interval:
+        default: 5
```

Comment:
Per the schema these are all invalid even though they functionally work:
- `description` is required
- `registry-username:` and the other empty ones are invalid (adding description should resolve)
- Apparently `default` has to be a string...

Reply:
this is resolved in the common tasks repo now